### PR TITLE
Add ips to OAUTH_SCOPES

### PR DESCRIPTION
**Pull request by rmcintosh**

### CODE REVIEW

**Comment by eatonphil** [comment]
Could have sworn I added this in there one time... Thanks!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "linode-manager",
-  "version": "0.9.4",
+  "version": "0.9.5",
```
Comment (eatonphil) [inline package.json]: Did you run `npm --no-git-tag-version version patch`? Another file would have been updated if so -- a shrinkwrap file.

**Comment by rmcintosh** [comment]
@eatonphil i did but neglected to commit that bit, somehow

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "linode-manager",
-  "version": "0.9.4",
+  "version": "0.9.6",
```
Comment (eatonphil) [inline package.json]: You should have reverted the change here before running that command. :) To fix: `git checkout origin/master -- package.json npm-shrinkwrap.json && npm --no-git-tag-version version patch`.

**Comment by eatonphil** [comment]
Alright sorry, you should rebase and run those same commands as last time to get the next version bump: 

`git checkout origin/master -- package.json npm-shrinkwrap.json && npm --no-git-tag-version version patch`

**Comment by rmcintosh** [comment]
lolwat

**Comment by eatonphil** [comment]
Thank you! This will be deployed shortly.

**Pull request merged**